js/packages/web/next.config.js

```diff
@@ -5,6 +5,9 @@ const dotenvLoad = require('dotenv-load');
 
 dotenvLoad();
 
+const debug = process.env.NODE_ENV !== 'production';
+const assetPrefix = !debug ? '/metaplex/' : '';
```

Comment:
can you move this to .env file

Reply:
ok, but you will have to specify the env before deploy building